src/plugins/visualizations/public/wizard/show_new_vis.tsx

```diff
@@ -94,7 +95,7 @@ export function showNewVisModal({
         </Suspense>
       );
     }),
-    { analytics: getAnalytics(), i18n: getI18n(), theme: getTheme() }
+    { analytics: getAnalytics(), i18n: getI18n(), theme: getTheme(), userProfile: getUserProfile() }
```

Comment:
A `getCore()` function isn't available in this plugin, and the `services` file already contains a ton of getters.